src/components/Feed/Feed.tsx

```diff
@@ -50,14 +52,19 @@ const Feed = ({ userSentiment, articles, selectedArticles, findMatchingArticles,
      })
 
     return (
-      <div className="articles-container">
-      <div className="all-sections">
-        <SectionForm findMatchingArticles={ findMatchingArticles }/>
-      </div>
-        <section className="articles-display">
-          { foundArticleCards.length ? foundArticleCards : articleCards }
-        </section>
-      </div>
+      <>
+        <div className="articles-container">
+          <Link to='/'>
+            <button className='retake-btn'>Retake Questionnaire</button>
```

Comment:
We need to add styling to the Retake Questionnaire, but maybe this will go in the styling branch.